.travis.yml

```diff
@@ -17,14 +17,16 @@ stages:
 
 jobs:
   include:
-    # test stage: run Node in all maintained versions. Run coverage only in the last
+    # test stage: run Node in all maintained versions. Run unique checks only in the last
     - node_js: 4
     - node_js: 6
     - node_js: 8
     - node_js: 9
       script:
         - npm run test
         - npm run coverage:upload
+        - npm run build:ts
+        - npm run validate:dts
```

Comment:
I think I would move `npm run validate:dts` into `postbuild:ts`, that way it will run in the release stage before the publish happens via semantic-release and it will be run automatically if people run `npm run npm run build:ts` locally.

Fixing a problem with the typescript definitions requires a new release, so it will be the right place to do it. What do you think?

Reply:
I was thinking this is more like a CI check, specifically because the last regression was due to making a change in the shared code for flow and TS and not propagating the changes to the typescript template. 

Now it would fail in that PR, instead of during a deploy later